Lib/glob.py

```diff
@@ -346,12 +223,30 @@ def lexists(path):
         """
         raise NotImplementedError
 
+    @staticmethod
+    def lstat(path, dir_fd=None):
+        """Implements os.lstat()
+        """
+        raise NotImplementedError
+
+    @staticmethod
+    def opendir(path, flags, dir_fd=None):
+        """Implements os.open()
+        """
+        raise NotImplementedError
+
     @staticmethod
     def scandir(path):
         """Implements os.scandir().
         """
         raise NotImplementedError
 
+    @staticmethod
+    def closedir(fd):
```

Comment:
Is the `close` method already taken? If not, I would keep the same name for future compatibility (it is weird that the only method that implementing some `os.*` operation has a different name). If it is not possible, I would use `closefd` (to stress that you expect a file descriptor object).